Orange/util.py

```diff
@@ -3,20 +3,25 @@
 from functools import wraps
 from itertools import chain, count
 from collections import OrderedDict
-import logging
+import warnings
 
 import numpy as np
 
 
-log = logging.getLogger()
+warnings.simplefilter('default', DeprecationWarning)
```

Comment:
Would filtering all DeprecationWarnings (once per line) actually be undesirable?